sha256block_amd64.go

```diff
@@ -1,4 +1,5 @@
-//+build !noasm,!appengine,gc
+//go:build !noasm && !appengine && gc
+// +build !noasm,!appengine,gc
```

Comment:
You only need the `//go:build` tag.

Reply:
I think not in this case as the min go version sha256-simd supports is 1.13.